pandas/tests/dtypes/test_inference.py

```diff
@@ -568,6 +568,13 @@ def test_maybe_convert_objects_nullable_integer(self, exp):
 
         tm.assert_extension_array_equal(result, exp)
 
+    def test_maybe_convert_objects_bool_nan(self):
```

Comment:
can you also add a value_counts tests in the OP (parametrize on dropna)

Reply:
Thanks! Additional tests added.